src/main/java/org/jenkinsci/plugins/fodupload/polling/ScanStatusPoller.java

```diff
@@ -68,7 +68,7 @@ public PollReleaseStatusResult pollReleaseStatus(final int releaseId) throws IOE
         
         if (analysisStatusTypes != null) {
             for (LookupItemsModel item : analysisStatusTypes) {
-                if (item.getText().equalsIgnoreCase(AnalysisStatusTypeEnum.Completed.name()) || item.getText().equalsIgnoreCase(AnalysisStatusTypeEnum.Canceled.name()))
+                if (item.getText().equalsIgnoreCase(AnalysisStatusTypeEnum.Completed.name()) || item.getText().equalsIgnoreCase(AnalysisStatusTypeEnum.Canceled.name()) || item.getText().equalsIgnoreCase(AnalysisStatusTypeEnum.InProgress.name()))
```

Comment:
So now if the scan is in progress, we mark it as complete?   I may not understand the context here.

Reply:
RIght now, in progress scanning would have already crashed before it got here. This addresses an edge case where the api tells it to wait. Eric asked for inProgressScanning to end the polling semi-gracefully because it could potentially sit for weeks. Even if an in progress scan is queued, the jenkins pipeline shouldn't wait.